src/averageAges.js

```diff
@@ -15,11 +15,14 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const filteredPeople = (century)
+    ? people.filter(person => Math.ceil(person.died / 100) === century
+      && person.sex === 'm')
+    : people.filter(person => person.sex === 'm');
```

Comment:
it can be simplified. the `centrury` check can be used inside the `filter` method.
please fix other cases and simplify them as well.
```suggestion
  const filteredPeople = people.filter(person => person.sex === 'm' 
    && century ? Math.ceil(person.died / 100) === century : true);
```